packages/material-ui/src/Select/SelectInput.js

```diff
@@ -282,15 +285,15 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         tabIndex={tabIndex}
         role="button"
         aria-expanded={open ? 'true' : undefined}
```

Comment:
Side question, would it work with?
```suggestion
        aria-expanded={open}
```

(smaller)

Reply:
It's explicitly defined that way in the apg:

> Set by the JavaScript when the listbox is displayed.
Otherwise, is not present.

Though it is quite at odds with how `aria-expanded` is defined 

> undefined (default) | The element, or another grouping element it controls, is neither expandable nor collapsible; all its child elements are shown or there are no child elements.
-- | --

I would wait for https://github.com/w3c/aria-practices/issues/618 to be resolved.

Reply:
Benchmarking for the combobox, I have found most implementation to set false. For instance: https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html.

Maybe more relevant:
https://www.w3.org/TR/wai-aria-1.1/#valuetype_true-false-undefined

![Capture d’écran 2019-10-19 à 13 18 29](https://user-images.githubusercontent.com/3165635/67144081-f7da8e80-f272-11e9-99ad-436c2da0e804.png)

Reply:
As I said: The original APG section unsets it and the regarding issue is not resolved. Same goes for the issue what a `select size="1"` should be mapped to (listbox vs combobox).